host_core/test/host_core/actors_test.exs

```diff
@@ -215,6 +215,10 @@ defmodule HostCore.ActorsTest do
   end
 
   test "can invoke via call alias" do
+    # TODO
+    # there's no way we should need this.
+    Process.sleep(500)
```

Comment:
Can we remove this and everything works still?

Reply:
Good catch